jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -963,6 +972,19 @@ private static final MethodHandle resolveInvokeDynamic(Class<?> clazz, String na
 		return result;
 	}
 	
+	/*
+	 * Retrieve the class name of the constant pool class element located at the specified
+	 * index in clazz's constant pool. Then, throw a NoClassDefFoundError with the cause
+	 * set as ClassNotFoundException. The message of NoClassDefFoundError and 
+	 * ClassNotFoundException contains the name of the class, which couldn't be found.
```

Comment:
Can you add javadoc style comment for the arguments / return types?

Reply:
fixed